src/Uno.UI/UI/Xaml/DragDrop/DropUITarget.cs

```diff
@@ -116,7 +102,7 @@ public IAsyncOperation<DataPackageOperation> DropAsync(CoreDragInfo dragInfo)
 			var target = VisualTreeHelper.HitTest(
 				dragInfo.Position,
 				xamlRoot,
-				getTestability: GetDropHitTestability,
+				isForDrop: true,
```

Comment:
This reduce the flexibilty of the `HitTest` method, not sure about the gain?

Reply:
@dr1rrb I can try to refactor it back to gain a bit of flexibility back